student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom = 'Unknown')
+    super()
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly notice that you should send the `age, name, parent_permission` in the `student` class to be able to send them in `super` to the `person` class.

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```

Reply:
Thank you so much @ShahierNashaat